packages/skeleton/src/plugin/themes/legacy/gold-nouveau.ts

```diff
@@ -1,4 +1,4 @@
-import type { PresetTheme } from '../index.js';
+// import type { Theme } from '../index.js';
 
 const goldNouveau = {
```

Comment:
I think an extension of Theme that had lines 6 through 19 would be a good idea, so that the themes are typechecked

Reply:
I don't think GitHub is aligning this message in the right place, but if you're referring to the `properties_dark` and `enhancements` - those aren't carrying over to v3!